Sources/TuistSupport/Utils/Dotenv.swift

```diff
@@ -0,0 +1,199 @@
+import Darwin
+import Foundation
+
+/// Structure used to load and save environment files.
+@dynamicMemberLookup
+public enum Dotenv {
```

Comment:
Either I'm missing something, or this contains much more than what we need for tuist, can we just have a simple `loadEnvFile` function that is tailored to our needs?